### PR TITLE
ADCM-1374 Added links Task object to Job, and Action object to Task

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

File: adcm_client/objects.py [hunk 1]
```diff
@@ -729,6 +740,15 @@ class Task(BaseAPIObject):
...
     url = None
+    object_type = None
+
+    @min_server_version('2020.08.27.00')
+    def action(self) -> "Action":
+        try:
+            return TASK_PARENT[self.object_type](
+                self._api, id=self.selector[self.object_type]).action(id=self.action_id)
+        except KeyError:
+            raise DoesNotExistAction from None
```
Comment (acmnu) [inline adcm_client/objects.py:751]: I don't like this name. It is better to call it UnknownTaskParent or something like this.

**Review by acmnu — CHANGES_REQUESTED** [review]

File: adcm_client/objects.py [hunk 2]
```diff
@@ -729,6 +740,15 @@ class Task(BaseAPIObject):
     selector = None
     status = None
     url = None
+    object_type = None
+
+    @min_server_version('2020.08.27.00')
+    def action(self) -> "Action":
+        try:
+            return TASK_PARENT[self.object_type](
+                self._api, id=self.selector[self.object_type]).action(id=self.action_id)
```
Comment (acmnu) [inline adcm_client/objects.py:749]: Use object_id for that instead of selector.

**Review by acmnu — CHANGES_REQUESTED** [review]

File: adcm_client/objects.py (hunk 2 above)
Comment (a-alferov) [reply adcm_client/objects.py:749]: Ok

**Comment by dgusakov** [comment]
bender build

**Comment by dgusakov** [comment]
bender build

**Comment by dgusakov** [comment]
bender build

**Review by acmnu — CHANGES_REQUESTED** [review]
> E1135, E1136

What?

**Review by dgusakov — APPROVED** [review]

**Comment by acmnu** [comment]
Part related to ADCM-1373 has been reverted because it is not related to the PR.

**Review by acmnu — APPROVED** [review]

**Pull request merged**